deployment/data-clean-room/deploy.sh

```diff
@@ -22,14 +22,10 @@ fi
 VAR_FILE=$(realpath $VAR_FILE)
 source $VAR_FILE
 
+namespace=$1
+
```

Comment:
let's also check if `$1` exists here. Otherwise it should bail out, correct?

Reply:
done